README.md

```diff
@@ -1035,10 +1044,19 @@ paragraphs()
 ```ts
 import { text } from '@ngneat/falso';
 
-// Architecto et modi. Dolorem delectus quod eos facere architecto tempore.
+// Fugiat qua
 text();
+
+// Unde doloribus maiores vel ipsum, occaecati a omni
+text({ charCount: 50 });
 ```
 
+Options:
+
+| Property               | Type   | Description                                                                             |
+| ---------------------- | ------ | --------------------------------------------------------------------------------------- |
+| charCount _(optional)_ | number | This takes an integer to set the total characters in the returned string (Min value: 1) |
```

Comment:
@theryansmee Do we also want to mention the max-value here?

Reply:
@Technologeek I removed the max value 😊